jerry-core/parser/js/js-parser-internal.h

```diff
@@ -118,9 +118,9 @@ typedef struct
  * All data allocated by the parser is
  * stored in parser_data_pages in the memory.
  */
-typedef struct parser_mem_page_t
+typedef struct parser_mem_page_s
```

Comment:
I would rather keep these. The _s suffix is very unusual to me. Furthermore in C struct x; and typedef ... x; are different types.